aviary/subsystems/propulsion/engine_scaling.py

```diff
@@ -2,10 +2,36 @@
 import openmdao.api as om
 
 from aviary.utils.aviary_values import AviaryValues
+from aviary.subsystems.propulsion.utils import EngineModelVariables
 from aviary.variable_info.functions import add_aviary_input
 from aviary.variable_info.variables import Aircraft, Dynamic, Mission
 
 
+MACH = EngineModelVariables.MACH
+ALTITUDE = EngineModelVariables.ALTITUDE
+THROTTLE = EngineModelVariables.THROTTLE
+HYBRID_THROTTLE = EngineModelVariables.HYBRID_THROTTLE
+THRUST = EngineModelVariables.THRUST
+TAILPIPE_THRUST = EngineModelVariables.TAILPIPE_THRUST
+GROSS_THRUST = EngineModelVariables.GROSS_THRUST
+SHAFT_POWER = EngineModelVariables.SHAFT_POWER
+SHAFT_POWER_CORRECTED = EngineModelVariables.SHAFT_POWER_CORRECTED
+RAM_DRAG = EngineModelVariables.RAM_DRAG
+FUEL_FLOW = EngineModelVariables.FUEL_FLOW
+ELECTRIC_POWER = EngineModelVariables.ELECTRIC_POWER_IN
+NOX_RATE = EngineModelVariables.NOX_RATE
+TEMPERATURE = EngineModelVariables.TEMPERATURE_T4
+# EXIT_AREA = EngineModelVariables.EXIT_AREA
+
+# these variables are not outputs or are variables that should not get scaled
+skip_variables = [MACH, ALTITUDE, THROTTLE, HYBRID_THROTTLE, TEMPERATURE]
+# these variables have a 'max' counterpart that should be added if they are present
+max_variables = {
+    THRUST: Dynamic.Mission.THRUST_MAX,
+    SHAFT_POWER: Dynamic.Mission.SHAFT_POWER_MAX,
+}
```

Comment:
looks like these might be used in a couple places, should we define them in one place and import them in the others?

Reply:
I forgot these got reused! I will move them to the propulsion utils